examples/misc/legends/legend-base.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
```

Comment:
All legend examples need to have zoom controls. I think we could add them in a separated issue, where we add some legend examples in the dev center. What do you think?